### PR TITLE
[MODULAR][QoL] Removes empty Hypovials from standard Hypospray Kits

### PR DESCRIPTION
**Pull request by softcerv**
## About The Pull Request
Removes the empty vials that come standard with the standard hypospray kits, including the one inserted into the hypospray.

## How This Contributes To The Skyrat Roleplay Experience
Since Hypovials are printable from the chem-master now, having empty hypovials within the kit introduces unnecessary tedium and things to discard. 

## Changelog

:cl:
qol: removes empty Hypovials from Hypospray Kits
/:cl:

### CODE REVIEW

**Comment by lessthnthree** [comment]
So instead to fill from plumbing factories we'll then carry bottles to the chemmaster, to turn into vials, adding tedium.

**Comment by softcerv** [comment]
> So instead to fill from plumbing factories we'll then carry bottles to the chemmaster, to turn into vials, adding tedium.

Most people end up discarding the empty hypovials on the floor because they use the chem-master to make new ones when making their mixes. From what I have seen, it is very rare that the included empty hypovials actually get used.

**Comment by Proximitron** [comment]
As someone who plays medical often:
I usually take the hypospray out, take the empty vial out of the hypospray and discard the hypospray kit altogehter, putting the hypospray in my belt.
Others i know keep the kit but discard all the empty vials. The reason for this is very simple: The empty vials are not labelt, while the ones from the chemmaster are labled automaticly on production. As the empty vials are insufficient for every medicin you need anyway, there is really no need for them.

File: modular_skyrat/modules/hyposprays/code/hyposprays_II.dm [hunk 1]
```diff
@@ -28,8 +28,8 @@
 	//Inject or spray?
 	var/mode = HYPO_INJECT
 	var/obj/item/reagent_containers/glass/vial/vial
-	var/start_vial = /obj/item/reagent_containers/glass/vial/small
-	var/spawnwithvial = TRUE
+	var/start_vial
+	var/spawnwithvial = FALSE
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/hyposprays/code/hyposprays_II.dm]: If you could document those variables while you're there, that'd be great.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Aside from the required documentation, I'd say this is a good change.

**Review by GoldenAlpharex — APPROVED** [review]
Now that's what I'm talking about!

**Pull request merged**